src/ogs.styl

```diff
@@ -256,6 +261,11 @@ dark.strong-loss                        = lighten(dark.loss, 25%)
 dark.weak-win                           = darken(dark.win, 25%)
 dark.weak-loss                          = darken(dark.loss, 25%)
 
+dark.strong-w-text                     = #3C3939
+dark.strong-l-text                     = #3C3939
```

Comment:
The contrast would be better with #1a1a1a (the site background color)

Reply:
updated to #1a1a1a